### PR TITLE
Fix typo in README, update CREDITS.md

### PR DESCRIPTION
**Pull request by zdimension**
**EDIT:** Since this PR consists of really small changes with not so descriptive commit messages, I would advise squashing it before merge

### CODE REVIEW

**Comment by MarcinOrlowski** [comment]
Isn't it "bug report" (singular) so plural is "bug reports" then?

**Comment by MarcinOrlowski** [comment]
Summoning @davidhutchens

**Comment by zdimension** [comment]
> Isn't it "bug report" (singular) so plural is "bug reports" then?

I've never seen `bugs reports` a single time in my life, only `bug report` (singular) and `bug reports` (plural). Here, the issues page is made for reporting bugs, so I think `bug reports` is the appropriate form (not a native English speaker though, might very well be wrong)

image

**Comment by MarcinOrlowski** [comment]
Staph! I read read your change as going from "bug reports" to "bugs report", not the other way around. And yes, "bugs reports" is wrong.

**Comment by davidhutchens** [comment]
I concur. This PR and "bug reports" are correct.

**Comment by zdimension** [comment]
> Staph! I read read your change as going from "bug reports" to "bugs reports", not the other way around. And yes, "bugs reports" is wrong. My bad. Please continue :)

No worries, happens to the best of us 😄 Since I'm on it, would you like me to add your name to the credits.md file? Your recent contributions to the project definitely qualify you as an "active contributor"

**Comment by MarcinOrlowski** [comment]
That would be nice. Thanks

File: docs/credits.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ https://www.pucminas.br/destaques/Paginas/default.aspx), Brazil
 * David H. Hutchens - [Millersville University](https://www.millersville.edu/), Pennsylvania, USA
 * Tom Niget - [LEAT](https://leat.univ-cotedazur.fr/), [Polytech Nice-Sophia](https://polytech.univ-cotedazur.fr/),
 France
+* Marcin Orlowski, USA
```
Comment (MarcinOrlowski) [inline docs/credits.md]: Can I have it in that form please:

[Marcin Orlowski](http://MarcinOrlowski.com), Poland

Comment (zdimension) [reply docs/credits.md]: Oops, sorry, saw Palo Alto on your profile and assumed USA would be the right one. Fixing it right now

**Review by maehne — APPROVED** [review]

**Pull request merged**